source/monitoring.txt

```diff
@@ -12,8 +12,135 @@ Monitoring and Logging
    :depth: 2
    :class: singlecol
 
+Overview
+--------
 
+Learn how to observe the behavior of your MongoDB source or sink
+connector through **monitoring**.
+Monitoring is the process of getting information about the
+activities a running program performs for use in an application
+or an application performance management library.
 
+To learn how monitoring works in the connector and how to use
+it, see the :ref:`<kafka-monitoring-information>` section.
+
+To view an example that shows how to monitor a running connector,
+see the :ref:`<kafka-monitoring-example>` section. 
+
+To view a list of all metrics produced by MongoDB source and sink
+connectors, see the :ref:`<kafka-monitoring-all-attributes>` section.
+
+To learn more about logging in the connector, see the
+:ref:`<kafka-monitoring>` guide.
+
+.. _kafka-monitoring-information:
+
+Monitor the Connector
+---------------------
+
+The {+mkc+} uses **{+jmx-long+} (JMX)** to enable monitoring.
+JMX is a technology included in the {+java-se+} that provides
+tools to monitor applications and devices. You can view the
+metrics produced by the connector with any JMX console, such
+as ``jconsole``.
+
+The {+mkc+} provides metrics for individual connector **tasks**.
+Tasks are classes instantiated by a connector that copy data to and
+from data stores and {+ak+}. A connector manages one or more tasks.
+
+To learn more about JMX, see the following resources from Oracle:
+
+- `Java Management Extensions Guide <https://docs.oracle.com/en/java/javase/17/jmx/introduction-jmx-technology.html>`__
+- `Using JConsole <https://docs.oracle.com/en/java/javase/17/management/using-jconsole.html>`__
+
+To learn more about tasks in {+kc+}, see the following resources:
+
+- `Kafka Connect Concepts from Confluent <https://docs.confluent.io/platform/current/connect/concepts.html#tasks>`__
+- `{+kc+} API documentation for Task interface <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/connector/Task.html>`__
+
+Enable Monitoring
+~~~~~~~~~~~~~~~~~
+
+The {+mkc+} uses {+kc+}'s metrics infrastructure to
+serve metrics. To read the metrics produced by your connector,
+enable JMX and launch your {+kc+} deployment.
+
+To learn how to enable JMX for a {+kc+} instance
+running on your host machine, see the
+`Official Kafka Documentation <https://kafka.apache.org/documentation/#monitoring>`__.
+
+To learn how to enable JMX for a containerized {+kc+}
+deployment, see
+`Kafka Monitoring and Metrics Using JMX with Docker <https://docs.confluent.io/platform/current/installation/docker/operations/monitoring.html>`__.
+
+.. _kafka-monitoring-example:
+
+Example - Monitor the Quick Start
```

Comment:
This example requires additional setup as the required connector is not yet published. If you wish to test the example, let me know and I can help get it working.